doc/source/reference/random/index.rst

```diff
@@ -7,6 +7,73 @@
 Random sampling (:mod:`numpy.random`)
 =====================================
 
+NumPy facilitates various tools for generating pseudo-random numbers. This
+also includes providing means to create repeatable sequences for reproducible
+pseudo-random processes. As NumPy random module has undergone substantial changes
+first the quick reference for the major differences are laid out to demonstrate
+what the usage style used to be pre NumPy v1.17 and what currently is for
+the impatient reader.
+
+.. _random-quick-start:
+
+Quick Reference
+---------------
+
+If you want to draw samples from various distributions, below is probably what you
+see in the old docs and elsewhere, roughly speaking, import ``random`` module and then
+use functions from the :ref:`legacy` section, such as
+
+.. code-block:: python
+
+  # pre-NumPy 1.17, nostalgic style,
```

Comment:
```suggestion
  # pre-NumPy 1.17
```
This is just a bit of a nitpick, but I don't think "nostalgic style" should be there.

Reply:
I agree. Will delete it.